### PR TITLE
VIH-6686 not show popup on first enter in judge wr

### PR DESCRIPTION
**Pull request by valentinabennett**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-6686

### Change description ###
Do not show change device popup on first entrance to the judge waiting room, show only on demand

**Does this PR introduce a breaking change?** (check one with "x")

```
[ X] Yes
[ ] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb.AcceptanceTests/Features/WaitingRoom.feature [hunk 1]
```diff
@@ -18,7 +18,6 @@ Scenario: Participant waiting room
 Scenario: Judge waiting room
 	Given the Judge user has progressed to the Waiting Room page
 	Then the user is on the Waiting Room page
-  And the judge dismisses the change camera popup
 	And a phone number for help is provided
```
Comment (npoxon) [inline VideoWeb/VideoWeb.AcceptanceTests/Features/WaitingRoom.feature:21]: Can you remove this from the test is Accessibility.feature as well please?

File: VideoWeb/VideoWeb.AcceptanceTests/Steps/WaitingRoomSteps.cs [hunk 2]
```diff
@@ -50,15 +50,6 @@ public void ThenTheFirstParticipantStatusIsDisplayedAsNotSignedIn(string name, s
...
 
-        [When(@"the judge dismisses the change camera popup")]
-        [Then(@"the judge dismisses the change camera popup")]
-        public void TheJudgeDismissesTheChangeCameraPopup()
-        {
-            _browsers[_c.CurrentUser].Driver.WaitUntilVisible(JudgeWaitingRoomPage.ChangeDeviceButton).Displayed.Should().BeTrue();
-            _browsers[_c.CurrentUser].Click(JudgeWaitingRoomPage.ChangeDeviceButton);
-            _browsers[_c.CurrentUser].Driver.WaitUntilElementNotVisible(JudgeWaitingRoomPage.ChangeDeviceButton);
-        }
-
```
Comment (npoxon) [inline VideoWeb/VideoWeb.AcceptanceTests/Steps/WaitingRoomSteps.cs:61]: Are you removing this completely? Is there no option to change the camera anymore?

File: VideoWeb/VideoWeb.AcceptanceTests/Pages/ClerkWaitingRoomPage.cs [hunk 3]
```diff
@@ -14,6 +14,5 @@ public static class JudgeWaitingRoomPage
         public static By PausedText = CommonLocators.ElementContainingText("Hearing paused");
         public static By ResumeVideoCallButton = CommonLocators.ButtonWithInnerText("Resume video hearing");
         public static By StartVideoHearingButton = CommonLocators.ButtonWithInnerText("Start video hearing");
-        public static By ChangeDeviceButton = By.Id("change-device-btn");
```
Comment (npoxon) [inline VideoWeb/VideoWeb.AcceptanceTests/Pages/ClerkWaitingRoomPage.cs:17]: Does the button no longer exist?

**Review by npoxon — CHANGES_REQUESTED** [review]

File: VideoWeb/VideoWeb.AcceptanceTests/Features/WaitingRoom.feature (hunk 1 above)
Comment (valentinabennett) [reply VideoWeb/VideoWeb.AcceptanceTests/Features/WaitingRoom.feature:21]: Removed

File: VideoWeb/VideoWeb.AcceptanceTests/Steps/WaitingRoomSteps.cs (hunk 2 above)
Comment (valentinabennett) [reply VideoWeb/VideoWeb.AcceptanceTests/Steps/WaitingRoomSteps.cs:61]: It's does not show on first entrance anymore, but should be display popup on press button.

**Review by RocheL — APPROVED** [review]

**Review by npoxon — APPROVED** [review]

**Pull request merged**